contentcuration/contentcuration/frontend/channelList/views/Channel/ChannelItem.vue

```diff
@@ -5,8 +5,8 @@
     :class="{hideHighlight}"
     data-test="channel-card"
     tabindex="0"
-    :to="channelRoute"
     :href="channelHref"
+    @click="goToChannelRoute"
```

Comment:
this may cause a regression of https://github.com/learningequality/studio/issues/2156

Reply:
The `href` is still there, but I `e.preventDefault()` the click. Do you know if that would still have the same a11y implications or does the existence of the `href` attribute suffice? I almost removed it but thought it might have a11y uses.